dev-libs/openssl/metadata.xml

```diff
@@ -1,7 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<maintainer>
-		<email>maintainer-wanted@gentoo.org</email>
-	</maintainer>
+<herd>base-system</herd>
```

Comment:
this ebuild is not currently maintained by base-system, it's maintained by, uhm... us